anecdote/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: Anecdote
+
+## Overview
+
+Anecdote is an app designed to help you track, prioritize, and notify your team about bugs reported in customer feedback, ensuring you're always there to support your customers the moment they encounter an issue.
+
+Anecdote's AI scans all your customer feedback, finds and groups together bug reports, and then creates instant reports and alerts for you.
+
+**How does it work?**
+
+Anecdote can tell if feedback is about a bug just by understanding the words used (semantically). It also groups, summarizes, and spots trends in these bugs to help prioritize them better.
+
+We cover more than 80 sources, including:
+
+
+- **Public feedback** from places like App Stores, Google Play, Twitter, Reddit, Discord, G2, Google Maps, Instagram, and Facebook.
+- **Customer support** tools such as Zendesk, Intercom, Freshdesk, Gorgias, Kustomer, and HubSpot.
+- **Survey platforms** like SurveyMonkey, Typeform, Delighted, and SurveySparrow.
+
+The Anecdote dashboard within Datadog enables you to track where customers report bugs, assess the severity of these bugs, and identify which bugs are most frequent, helping you to prioritize efficiently.
+
+We also send alerts for specific bugs into Slack and email so that you’re alerted where you work.
```

Comment:
The overview should include: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What specific data your integration monitors and the value?

Reply:
Changed